pkg/thing/delivery/http/thing_proxy.go

```diff
@@ -22,6 +22,7 @@ func (err errorConflict) Error() string {
 type ThingProxy interface {
 	Create(id, name, authorization string) (idGenerated string, err error)
 	UpdateSchema(authorization, ID string, schemaList []entities.Schema) error
+	UpdateConfig(authorization, ID string, configList []entities.Config) error
```

Comment:
I'm thinking if that operation needs to change something in `updateSchema` too. For example, what will occur if the schema is updated after an `updateConfig`?

Reply:
Yes, some investigation should be done. But I think we can create a specific issue to cover that.

Reply:
BLI-626 was created to map this issue since it will involve discussion with other teams.